core/common/src/main/java/alluxio/wire/FileInfo.java

```diff
@@ -59,9 +61,9 @@
   private long mMountId;
   private int mInAlluxioPercentage;
   private String mUfsFingerprint = Constants.INVALID_UFS_FINGERPRINT;
-  // A list of String ACL entries, to support json serialization.
-  private ArrayList<String> mAclEntries = new ArrayList<>();
-  private ArrayList<String> mDefaultAclEntries = new ArrayList<>();
+
+  private AccessControlList mAcl = new AccessControlList();
```

Comment:
Looking at other parts of this file, it looks like these are allowed to be `null`. Why not use `null`?

Reply:
made the decision to not allow them to be `null`